templates/new_device_login.txt

```diff
@@ -5,8 +5,8 @@
 {{#if location}}{{ location }}{{/if}}
 {{#if timestamp}}{{ timestamp }}{{/if}}
 
-{{t "This is an automated email; if you didn't add a new device to your Firefox Account, you should change your password immediately at %(resetLink)s."}}
-{{t "For more information, please visit %(supportUrl)s"}}
+{{{t "This is an automated email; if you didn't add a new device to your Firefox Account, you should change your password immediately at %(passwordChangeLink)s."}}}
```

Comment:
I know the trailing period was there before, when copying the link out of the email users won't know whether that period should be there or not. If the user copies the link w/ the period, the utm_content value will be `fx-change-password.`

For reference, the utm_content is currently:

```
&utm_content=fx-change-password.
```